tractor/log.py

```diff
@@ -31,11 +31,13 @@
     'GARBAGE': 1,
     'TRACE': 5,
     'PROFILE': 15,
+    'RUNTIME': 500,
```

Comment:
Yeah, basically this will be the "logging we want users to actually see by default", above 'error' but below 'critical'. It's white and shouldn't be too invasive or alarming.